src/Framework/TargetBuiltReason.cs

```diff
@@ -28,6 +28,22 @@ public enum TargetBuiltReason
         /// <summary>
         /// The target was part of the parent's AfterTargets list.
         /// </summary>
-        AfterTargets
+        AfterTargets,
+
+        /// <summary>
+        /// The target was defined as an initial target of the project.
+        /// </summary>
+        InitialTarget,
+
+
+        /// <summary>
+        /// The target was the default target of the project
+        /// </summary>
+        DefaultTarget,
```

Comment:
Pluralization should match the xml attribute app DefaultTargets and InitialTargets like AfterTargets.
Also EntryTargets adds that can be several (/t:a;b)

Reply:
so, to clarify, all three new enum fields should be plural, right?

Reply:
Yes